lib/ClimaLandSimulations/src/utilities/pfts.jl

```diff
@@ -0,0 +1,614 @@
+    :f_root_to_shoot, # Root to Shoot Ratio
+    :K_sat_plant,     # Hydraulic Conductivity at Saturation
+    :ψ63,             # Xylem water potential at 63% loss of conductivity
+    :Weibull_param,   # Conductivity Weibull Parameter
+    :a,               # Retention Curve Curvature Parameter
+    :capacity,        # Water Storage Capacity
+    :plant_S_s,       # Specific Storativity
+    :rooting_depth    # Plant Rooting Depth
+]
+
+
+"""
+Define a PFT type that can be used to store the parameters for each PFT.
+Each PFT has a name and a list of parameters. The parameters are stored as 
+NamedTuples with the parameter name as the first element and the parameter value
+as the second element. The inner constructor checks to ensure all required
+parameters are defined for the PFT.
+"""
+struct Pft
+    name::String
+    parameters::NamedTuple
+
+    # Inner constructor to verify all required parameters are defined
+    function Pft(name, parameters)
+        defined_param_names = keys(parameters)
+        for param in param_list
+            if param ∉ defined_param_names
+                error("PFT $name is missing parameter $param")
+            end
+        end
+        new(name, parameters)
+    end
+end
+
+
+# The default PFTs used in the model use parameter values drawn from defferent 
+# sources in the literature. The sources are listed here with abbreviations that
+# are used in the comments in the code below.
+#
+#    Chen, J. M., G. Mo, J. Pisek, J. Liu, F. Deng, M. Ishizawa, 
+#    and D. Chan (2012), Effects of foliage clumping on the estimation of global
+#    terrestrial gross primary productivity, Global Biogeochem. Cycles, 26, 
+#    GB1019, doi:10.1029/2010GB003996.
+#    abb. as (Chen et al. 2012)
+#
+#    CLM5.0 Technical Note
+#    abb. as (CLM5.0)
+#
+#    Best, M. J., Pryor, M., Clark, D. B., Rooney, G. G., Essery, R. L. H., 
+#    Ménard, C. B., Edwards, J. M., Hendry, M. A., Porson, A., Gedney, N.,
+#    Mercado, L. M., Sitch, S., Blyth, E., Boucher, O., Cox, P. M., Grimmond, 
+#    C. S. B., and Harding, R. J.: The Joint UK Land Environment Simulator 
+#    (JULES), model description – Part 1: Energy and water fluxes, Geosci. Model
+#    Dev., 4, 677–699, https://doi.org/10.5194/gmd-4-677-2011, 2011.
+#    abb as (JULES Model Description) 
+#
+#    Oliver, Rebecca & Mercado, Lina & Clark, Doug & Huntingford,
+#    Chris & Taylor, Christopher & Vidale, P.L. & McGuire, Patrick & Todt, 
+#    Markus & Folwell, Sonja & Semeena, Valiyaveetil & Medlyn, Belinda. (2022).
+#    Improved representation of plant physiology in the JULES-vn5.6 land surface
+#    model: Photosynthesis, stomatal conductance and thermal acclimation. 
+#    10.5194/gmd-2022-11. 
+#    abb. as (Oliver et al. 2022)
+
+# Define the default PFTs that are used in the model.
+
+NET_Temp = Pft(
+    "Temperate Needleleaf Evergreen Tree",
+    (
+        # Radiative Transfer Model
+        Ω = 0.74,          # Chen et al. 2012 Table 1
```

Comment:
Something I was thinking about here is that our radiative transfer model doesn't distinguish between stem area and leaf area because the MODIS LAI contains both LAI and SAI, but we have been using the radiative transfer parameters from [CLM](https://escomp.github.io/ctsm-docs/versions/release-clm5.0/html/tech_note/Surface_Albedos/CLM50_Tech_Note_Surface_Albedos.html) for leaf area only and the stem values can vary quite widely from the leaf values. Perhaps we should be doing something with a linear combination of the steam and leaf values for each PFT? But then we don't really know the fraction of leaf to stem. Perhaps there is a characteristic stem to leaf ratio for each PFT and we could use this to weight the reflectances and transmittances?

Reply:
I think it is fine if we consider the optical depth is made only of LAI. Other LSMs do it without losing much.